response.go

```diff
@@ -586,6 +586,31 @@ func NewJsonResponse(status int, body any) (*http.Response, error) { // nolint:
 	return response, nil
 }
 
+// NewJsonResponseOrPanic is like [NewJsonResponse] but panics in case or error.
+//
+// It simplifies the call of [ResponderFromMultipleResponses], avoiding the use of a
+// temporary variable and an error check, and so can be used in such context:
+//
+//	httpmock.RegisterResponder(
+//	  "GET",
+//	  "/test/path",
+//	  httpmock.ResponderFromMultipleResponses([]*http.Response{
+//		httpmock.NewJsonResponseOrPanic(200, &MyFirstResponseBody),
```

Comment:
Only the first column should be a TAB, following columns should be 2 spaces per indent, like you did the line just above:
```suggestion
//	    httpmock.NewJsonResponseOrPanic(200, &MyFirstResponseBody),
```